Dockerfile

```diff
@@ -1,2 +1,10 @@
 FROM microsoft/dotnet:2.2-sdk
```

Comment:
better use the official image from dotnet sdk
mcr.microsoft.com/dotnet/core/sdk:2.2